### PR TITLE
refactor, chore(egen): Removes boilerplate, heading fixes, and other corrections from template

### PR DESCRIPTION
**Pull request by nileshspringml**
**REQUIRED:** 
Fixing below issues in notebook:
- Updated template of notebook based on new template of notebook
- Link of colab and github code is changed to correct link
- Tested code and its working fine

<br>
Changed notebook to updated template of notebook. Additionally, thoroughly checked notebook.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store_bigtable.ipynb [hunk 1]
```diff
@@ -130,205 +133,129 @@
...
       "cell_type": "markdown",
       "metadata": {
-        "id": "UzsMphY2t4-v"
+        "id": "54c5ef8a8f43"
       },
       "source": [
-        "**1. Vertex AI Workbench**\n",
-        "* Do nothing as you are already authenticated."
+        "<div class=\"alert alert-block alert-warning\">\n",
+        "<b>⚠️ The kernel is going to restart. Please wait until it is finished before continuing to the next step. ⚠️</b>\n",
```
Comment (kittyabs) [inline notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store_bigtable.ipynb:200]: remove "Please" and contract "it is":

- Wait until it's finished...

**Comment by katiemn** [comment]
Is this draft pull request ready for review?

**Comment by nileshspringml** [comment]
> Is this draft pull request ready for review?

Yes, ready for review

**Review by katiemn — APPROVED** [review]

File: notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store_bigtable.ipynb [hunk 2]
```diff
@@ -553,7 +455,7 @@
         "    * Create a `FeatureOnlineStore` instance with autoscaling.\n",
         "1. Define the data (`FeatureView`) to be served by the newly-created instance. This can either map to\n",
         "    * The BigQuery view that you just created for serving data.\n",
-        "    * The `FeatureGroup` and `Feature` we will create to host feature metadata.\n",
+        "    * The `FeatureGroup` and `Feature` you will create to host feature metadata.\n",
```
Comment (kittyabs) [inline notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store_bigtable.ipynb:458]: change "you will" to "you'll"

File: notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store_bigtable.ipynb [hunk 3]
```diff
@@ -768,7 +670,7 @@
       "source": [
         "#### [Optional] Create FeatureGroup/Features\n",
         "\n",
-        "Create a FeatureGroup pointing to the created BigQuery view for the demo. We will then create features for each column we would like to register.\n"
+        "Create a FeatureGroup pointing to the created BigQuery view for the demo. You will then create features for each column you would like to register.\n"
```
Comment (kittyabs) [inline notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store_bigtable.ipynb:673]: Change:
- " You will then create features for each column you would like to register"
to:
- " You then create features for each column you'd like to register"

**Review by kittyabs — COMMENTED** [review]
some suggested edits

File: notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store_bigtable.ipynb (hunk 3 above)
Comment (nileshspringml) [reply notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store_bigtable.ipynb]: Did the required changes

File: notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store_bigtable.ipynb (hunk 2 above)
Comment (nileshspringml) [reply notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store_bigtable.ipynb]: Did the required changes

**Comment by nileshspringml** [comment]
> some suggested edits

We implemented all the suggestion

**Pull request merged**